src/main/java/com/tratif/pdfgen/document/DocumentBuilder.java

```diff
@@ -52,4 +51,8 @@
             throw new RuntimeException("There was a problem with the file.", e);
         }
     }
+
+    public String getContent() {
```

Comment:
remove it -- we shouldn't add methods to production code only to be able to test it.